### PR TITLE
Use membership commons for zuora squish

### PR DESCRIPTION
**Pull request by chrisjowen**
Moving zuora specific code to membership common

### CODE REVIEW

**Comment by chrisjowen** [comment]
@tudorraul can you please review this today?

**Comment by mario-galic** [comment]
:+1: Digital subscription can be purchased as a registered user as well as a guest user. SF and Zuora reflect the purchase.

File: app/forms/SubscriptionsForm.scala [hunk 1]
```diff
@@ -26,12 +27,13 @@ object SubscriptionsForm {
...
   private val emailMaxLength = 240
 
+  def unapplyUkAddress(a: Address) = Some((a.lineOne, a.lineTwo, a.town, a.postCode))
   val addressDataMapping = mapping(
     "address1" -> text(0, addressMaxLength),
     "address2" -> text(0, addressMaxLength),
     "town" -> text(0, addressMaxLength),
     "postcode" -> text(0, addressMaxLength)
-  )(AddressData.apply)(AddressData.unapply)
+  )(Address.apply(_:String, _:String, _: String, "United Kingdom", _: String, Countries.UK))(unapplyUkAddress)
```
Comment (tudorraul) [inline app/forms/SubscriptionsForm.scala]: why is there UK present in different formats here?

Comment (tudorraul) [inline app/forms/SubscriptionsForm.scala]: also, can this definition of apply be avoided?

Comment (chrisjowen) [inline app/forms/SubscriptionsForm.scala]: The string "United Kingdom" is wrong in this context, it maps the form data to an Address that has "United Kingdom" as a countyOrState property.

**Comment by tudorraul** [comment]
:+1:

**Comment by rtyley** [comment]
:+1:

**Comment by chrisjowen** [comment]
Tested on prod
image

**Comment by chrisjowen** [comment]
image

**Pull request merged**